<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidgetUI">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>英雄</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>英雄介绍</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>德玛</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>上单</string>
      </property>
      <property name="text">
       <string>攻击防厚</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>忙升</string>
     </property>
     <item>
      <property name="text">
       <string>打野</string>
      </property>
      <property name="text">
       <string>战士宫高</string>
      </property>
     </item>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBoxMarry">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>131</width>
      <height>63</height>
     </rect>
    </property>
    <property name="title">
     <string>婚否:</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="btnYh">
       <property name="text">
        <string>已婚</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="btnWh">
       <property name="text">
        <string>未婚</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxSex">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>119</width>
      <height>69</height>
     </rect>
    </property>
    <property name="title">
     <string>性别:</string>
    </property>
    <widget class="QRadioButton" name="btnFemale">
     <property name="geometry">
      <rect>
       <x>64</x>
       <y>33</y>
       <width>39</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>女</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="btnMale">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>33</y>
       <width>39</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>男</string>
     </property>
    </widget>
   </widget>
   <widget class="QTreeWidget" name="treeWidgetCode">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidgetUi">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>英雄</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>性别</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>年龄</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>关羽</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>男</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>张飞</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>男</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>19</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tableWidgetCode">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>480</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hello World!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>pic</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_movie">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>351</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>gif play label</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_paly">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>430</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>430</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>停止播放</string>
    </property>
   </widget>
   <widget class="MyWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>-10</y>
      <width>298</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_getApi">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>获取进度条</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setApi">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>设置99</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyWidget</class>
   <extends>QWidget</extends>
   <header location="global">mywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>btnMale</tabstop>
  <tabstop>btnFemale</tabstop>
  <tabstop>treeWidgetUI</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
